### DISCUSSION TITLE
Problem loading 2 times the same model

### DISCUSSION THREAD

**Discussion by LuginiAndrea** [enable3d/enable3d#143] [Help / Questions] [answered]
Hello, I encountered a problem when i want to load twice the same model.
The given error is: "Uncaught (in promise) TypeError: t.lastIndexOf is not a function".

The "incrimaniting" code is the following:
```
function addBook(mainScene) {
  let model = new ExtendedObject3D()
  console.log("Here")
  mainScene.load.gltf("../Resources/Models/Inventory/scene.gltf").then(gltf => {
    console.log("Here 2")
    let child = gltf.scene.children[0]
    child.scale
    model.add(child)
  })
  return model
}
```

The error is given inside the load.gltf function 

Any ideas why?

**Comment by yandeu** [comment, marked as answer]
Same as #115
